### DISCUSSION TITLE
Next js with auth0

### DISCUSSION THREAD

**Discussion by KetulCodiste** [Web3Auth/Web3Auth#503] [Help]
I am trying to integrate the auth0 with nextjs app.
I have created a project in auth0, used the auth0 clientId and auth0 domain to create the verifier and used the following code in the frontend to get the instance of web3Auth:

`const web3auth = new Web3AuthCore({
          chainConfig: {
            chainNamespace: CHAIN_NAMESPACES.EIP155,
            rpcTarget: "https://rinkeby.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161",
            blockExplorer: "https://rinkey.etherscan.io",
            chainId: "0x4",
            displayName: "Rinkeby",
            ticker: "ETH",
            tickerName: "Ethereum",
          },
        })`

But this code is giving me the following error:
WalletInitializationError: Invalid params passed in, Please provide a valid clientId in constructor

The strange thing is that, this piece of code is working fine in the react demo, but giving error in next js demo
The difference in both these application is that in react js, i have override some webpack configuration and in next js I did not. I tried to override in next js as well but the error did not go, so deleted the override file. other difference is that in react js, I am using rewired and running the scripts using "react-app-rewired start" and in next js "next dev".

**Comment by himanshuchawla009** [comment]
can you confirm the version of web3auth you are using, please use the latest version 2.0.2. 

Also version >=2.0.0 is breaking change for @web3auth/core package, you need to pass clientId in  `Web3AuthCore` constructor.

**Reply by KetulCodiste** [reply]
Yes, it was a version issue. Thank you
